zfs/zfs-utils/PKGBUILD

```diff
@@ -0,0 +1,87 @@
+# Maintainer: Eric Naim <dnaim@cachyos.org>
+# Contributor: Kevin Stolp <kevinstolp@gmail.com>
+# Contributor: Eli Schwartz <eschwartz@archlinux.org>
+# Contributor: Iacopo Isimbaldi <isiachi@rhye.it>
+
+pkgname=zfs-utils
+pkgver=2.3.2
+pkgrel=1
+pkgdesc="Userspace utilities for the Zettabyte File System."
+arch=("i686" "x86_64" "aarch64")
+url="https://zfsonlinux.org/"
+license=('CDDL')
+optdepends=('python: for arcstat/arc_summary/dbufstat')
+source=("https://github.com/zfsonlinux/zfs/releases/download/zfs-${pkgver}/zfs-${pkgver}.tar.gz"{,.asc}
```

Comment:
Can we change here to use git and maybe pass it to the same commit as we have in the kernel?